src/styles/main.scss

```diff
@@ -1,3 +1,13 @@
+@import "_variables.scss";
+@import "_calendar.scss";
+
 body {
```

Comment:
Wszystko co dotyczy tutaj body powinno być w pliku _reset.scss

Reply:
Nie ma takiego punktu w instrukcji by stworzyć plik o takiej nazwie, a więc nie ma to znaczenia